docs/DeveloperGuide.md

```diff
@@ -452,6 +462,148 @@ facilitates the lending of money and ensures data integrity by validating input
 The TransactionList class provides essential functionalities for managing a list of transactions.
 Its methods facilitate the addition, removal, editing, and retrieval of transactions, ensuring efficient management of transactions within a group.
 
+### DateTime
+
+<ins> Overview </ins>
+
+The DateTime class handles all operations in LongAh involving the tracking of time. This includes storing and printing
+the datetime elements in dated transactions, parsing user's date & time related inputs as well as filtering transactions
+according to their stored date & time. Implementation of the class is made possible with the help of the *java.time*
+system class.
+
+<ins> Class Fields </ins>
+
+Storing requirements only occurs for the specific datetime component of the class. Hence, the class field structure is
+as follows:
+
+- *dateTime*: A dateTime object from *java.time* representing date & time associated with the current 
+instance.
+
+<ins> Constructor </ins>
+
+The DateTime constructor takes in a string representation of date & time in the `dd-MM-YYYY HHmm` form and parse it into
```

Comment:
Can look within our documents to check for standardisation of capital letters on this. `DD-MM-YYYY HHMM` seems more appropriate. Usually when characters are differentiated by captilisation it indicates that one is in numbers while the other is in words (i.e. dd-MMM-yyyy -> 15-Apr-2024)

Reply:
At the same time to look through our code, especially the error messages and ensure that it corresponds to this update.

Reply:
Resolved!